fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/listitem/ListItem.kt

```diff
@@ -285,14 +285,16 @@ object ListItem {
             Alignment.Bottom -> Alignment.BottomEnd
             else -> Alignment.CenterEnd
         }
+        val isClickable = onClick != null
```

Comment:
Can there be a situation when onClick is changed on runtime. If yes, then this line would not reflect the change as it won't recompose.